### ISSUE TITLE
[MSFT-245] [Registration] JAWS : Windows / Edge : Input field labels are announced twice

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2898]
**Steps to Reproduce**
1. Launch JAWS
2. Launch "https://accounts.topcoder.com/member/registration" in Windows / Edge
3. Navigate by virtual cursor [ARROW keys)
4. Navigate through input fields

**Expected Result**
Input fields' label should be announce only once

**Actual Result**
Note that each input field's label is announced twice continuously

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FJAWS%20Edge%20input%20field%20labels%20announced%20twice

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096233 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096233 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096233 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096233 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu Not working anymore in latest JAWS, can you please confirm?

**Comment by mishacucicea** [comment]
@asadath1395 What is not working anymore? Please be very specific.

**Comment by asadath1395** [comment]
The issue is not reproducible in latest JAWS

**Comment by mishacucicea** [comment]
@asadath1395 Seems like, tested on the latest `qa-accessibility branch`.

**Comment by mishacucicea** [comment]
@tosha5252 / @drasticdpk Can you confirm?

**Comment by asadath1395** [comment]
@mishacucicea I tested it on latest JAWS version and i couldn't reproduce

**Comment by mishacucicea** [comment]
@asadath1395 I agree with you, was just asking for confirmation from the testers.

**Comment by drasticdpk** [comment]
Hi @mishacucicea , since screen reader JAWS is showing unexpected behaviour for each time. If developer are unable to replicate the issue with JAWS screen reader , you can close the issue. 
Here is the reference link (Page is redirected after reading userName fields automatically)
https://monosnap.com/file/oQnRF1EH20LyrSuB2buucvpNgJfkye

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2898 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2898 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2898 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096749 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096749 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu @mishacucicea I think its better to close this one then, as per above comment

**Comment by mishacucicea** [comment]
Closing as can't reproduce.
